app/controlplane/pkg/biz/authz.go

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2025 The Chainloop Authors.
```

Comment:
This is the main change.

It's basically a custom enforcer that knows not only about casbin but also about API-Tokens custom ACLs.

I created it in the `biz` layer since that way it can be used without the risk of package cycles.

I tried to do smth similar but by extending the current `pkg/authz` but that one is connected already referenced by both `biz` and `data`